src/model/entity/backpack.go

```diff
@@ -0,0 +1,11 @@
+package entity
+
+type Backpack struct {
+	Capacity       uint
+	ConsumablesNum uint
```

Comment:
`Consumable` - "расходный материал" (https://dictionary.cambridge.org/uk/dictionary/english/consumable). Не совсем корректное имя в том смысле, что в рюкзаке могут храниться не только "расходные" материалы, но и вполне себе "постоянные" - оружие и сокровища.

Лучше заменить на просто `ItemsNum` - и проще получается, и по значению лучше подходит.
```suggestion
type Backpack struct {
	Capacity       uint
	ItemsNum    uint
```